### ISSUE TITLE
keepass2android - huawei - the file can not safe because of restrictions in android 4.4+

### ISSUE THREAD

**Issue by Meza100** [PhilippC/keepass2android#1639]
Hi together,

I try to open the database in the Directory "KeepassXC" (in storage). As I click on ok, the following window pop up:

The gewählte Datei kann aufgrund von Beschränkungen in Android 4.4+ von Keepass2Android nicht geändert werden. Um sie zu bearbeiten, muss sie an einen anderen Speicherort kopiert werden. Tippe auf OK, um einen Speicherort zu wählen."

If I save the database in another folder, so the syncthing won't function. Do anyone has an idea?

**Comment by PhilippC** [comment]
can you please see if this is fixed in 1.09a-pre? https://play.google.com/apps/testing/keepass2android.keepass2android

**Comment by MichiFr** [comment]
No, unfortunately it's not solved. Seeingthe same popups from time to time with a Huawei MediaPad M5 and latest Firefox Android.
